pkg/canary/writer/writer.go

```diff
@@ -58,6 +65,10 @@ func (w *Writer) run() {
 		select {
 		case <-t.C:
 			t := time.Now()
+			if i := rand.Intn(99) + 1; i <= w.outOfOrderPercentage {
```

Comment:
```suggestion
			if i := rand.Intn(100) + 1; i <= w.outOfOrderPercentage {
```
I think this should be `Intn(100)` due to inclusivity (`[0,n)`).

Reply:
No you're right it should be `100`, I misread the notation. 

This rng is to decide whether the log line should be out of order in the first place, not how much time to subtract.